### ISSUE TITLE
AppLauncher binding misbehaving

### ISSUE THREAD

**Issue by rhjoerg** [MagicSmokeIndustries/InfernalRobotics#66]
Not using the ToolBar mod, I'm dependent the AppLauncher.

Right now, InfernalRobotics' icon overlaps with the core icons, shows twice or overlaps with something else.

(KSP 1.0.4, InfernalRobotics 0.21.3)

In KSC:
![screenshot1](https://cloud.githubusercontent.com/assets/13078373/8649155/9c5032b4-2967-11e5-8a30-1e35ee475fc3.png)

In VAB:
![screenshot0](https://cloud.githubusercontent.com/assets/13078373/8649179/d70958fe-2967-11e5-9eee-bce7d90e9970.png)

In Flight:
![screenshot2](https://cloud.githubusercontent.com/assets/13078373/8649210/3699df6e-2968-11e5-8296-39db3f3b9b56.png)

**Comment by ZiwKerman** [comment]
Ugh, this bug again. Please attach your log here (output_log.txt).

**Comment by rhjoerg** [comment]
I just had a hard time to reconstruct the bug.

Entered game: ok
Entered VAB, flight, revert to VAB, repeating, back to KSC: all ok
Entered Flight via Vessel selection by clicking on LaunchPad in KSC: bug shows up and stays.

I scanned shortly over IR's and StageRecovery's AppLauncher code (as in my setting, IR's icon is just right to SR's one), but couldn't find any obvious mistake. I did not follow all side effects of modifying static references or boolean flags though as I cannot concentrate enough right now,

Log:
https://rhjoerg.github.io/files/ksp_output_log_201507131450.txt
